app/routers/organizations.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter, status, HTTPException
+from database import client
+from models import Organization
+
+router = APIRouter()
+
+
+@router.get("/check-auth-token/{api_key}", response_model=Organization)
+async def get_auth_token(api_key: str):
```

Comment:
This is not really "get" auth token right?
Maybe something like "verify_auth_token"?